JeMPI_Apps/JeMPI_UI/src/components/dashboard/Dashboard.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  Box,
+  Container,
+  Grid,
+  Stack,
+  Tab,
+  Tabs,
+  Typography
+} from '@mui/material'
+import CountWidget from './widgets/CountWidgetWidget'
+import {
+  Layers,
+  NotificationAdd,
+  NotificationsOff,
+  Person
+} from '@mui/icons-material'
+import { pink } from '@mui/material/colors'
+import MandU from './widgets/MandUWidget'
+import BetaFscore from './widgets/BetaFscoreWidget'
+import ConfusionMatrix from './widgets/ConfusionMatrixWidget'
+import { useState } from 'react'
+
+interface TabPanelProps {
+  children?: React.ReactNode
+  index: number
+  value: number
+}
+
+const CustomTabPanel = (props: TabPanelProps) => {
+  const { children, value, index, ...other } = props
+  return (
+    <Box
+      component={'div'}
+      role="tabpanel"
+      hidden={value !== index}
+      id={`dashboard-tabpanel-${index}`}
+      aria-labelledby={`dashboard-tab-${index}`}
+      {...other}
+    >
+      {value === index && children}
+    </Box>
+  )
+}
+
+const tabProps = (index: number) => {
+  return {
+    id: `dashboard-tab-${index}`,
+    'aria-controls': `dashboard-tabpanel-${index}`
+  }
+}
+
+const Dashboard = () => {
+  const [currentTabIndex, setCurrentTabIndex] = useState(0)
+  const handleChangeTab = (event: React.SyntheticEvent, newValue: number) => {
+    setCurrentTabIndex(newValue)
+  }
+  return (
+    <Container maxWidth={false}>
+      <Stack
+        padding={{
+          lg: '0rem 1rem 1rem 1rem',
+          xs: '1rem 0rem 0rem 0rem',
+          backgroundColor: '#fff',
+          borderRadius: '1rem'
+        }}
+      >
+        <Tabs
+          value={currentTabIndex}
+          onChange={handleChangeTab}
+          sx={{
+            paddingY: { lg: '2rem', xs: ' 0.5rem' }
+          }}
+          aria-label="dashboard tabs"
+          variant="scrollable"
+          scrollButtons
+          allowScrollButtonsMobile
+        >
+          <Tab
```

Comment:
**Something to consider (not needed for now though)**: Maybe we should a for loop here
i.e 
```
{tabDetails.map((details, index) => {
<label ...> {details.label}</labe>
}) }

{tabDetails.map((details, index) => {
<CustomTabPanel ....>{detail.component}</CustomTabPanel>
}) }
}
```

At least with the above you wont have to worry about mapping the tab details, to the correct  content. This will be particularly useful if we ever end up having multiple/a lot tabs on the dashboard

Reply:
@walisc For now we have just 2, when data is ready we will create a more generic DRY  version.